src/main/java/frc/robot/subsystems/Leds.java

```diff
@@ -229,4 +269,54 @@ protected void interrupted() {
             }
         };
     }
+
+    public Command turnOnVisionLights() {
+        return new InstantCommand("turn on the color known as vision", this, () -> {
+            ldrive_can.SetColor(3, Color.GREEN, 1);
+        });
+    }
+
+    public Command turnOffVisionLights() {
+        return new InstantCommand("turn off the color known as vision", this, () -> {
+            ldrive_can.SetColor(3, Color.OFF, 1);
+        });
+    }
+
+    public Command blinkVisionLights(int frequency) {
+        return new Command("Blink Vision Lights", this) {
+            int counter = 0;
+            boolean lightsOn = true;
+
+            @Override
+            protected void initialize() {
+                counter = 0;
+                ldrive_can.SetColor(3, Color.GREEN, 1.0);
```

Comment:
Would you kindly make this 3 a constant up above so we know which bank it represents?

Reply:
done, named it visionBank

Reply:
done